samples/SampleApp.Tests/HttpServerFixture.cs

```diff
@@ -51,6 +51,8 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
             builder.ConfigureLogging((p) => p.AddXUnit(this));
         }
 
+        #region interception-filter
```

Comment:
I _think_ I'd prefer the comment style to the region style.

Reply:
why? 

 * they enforce that every start has a matching end at compile time
 * you get intellisense for end region completion
 * there are shortcuts for navigating to the start or end of a region
 * u can disable the auto collapse https://github.com/SimonCropp/MarkdownSnippets/blob/master/docs/stop-regions-collapsing.png

Reply:
Because I'm just not a fan of regions in general, and I don't really want to have to manually reconfigure my IDE to not collapse regions just for a bit of documentation.

If MarkdownSnippet supports comments as a feature, I'd rather use that option than change around this repo to meet _its_ defaults.

Reply:
done